app/assets/javascripts/components/accept.js.jsx

```diff
@@ -0,0 +1,35 @@
+var AcceptButton = React.createClass({
+  getInitialState: function () {
+    return {
+      accepted: false
+    };
+  },
+//need URL to be whoever the invitee accepting is
+  handleClick: function (event) {
+    var invitee_id = this.props.inviteeID
+      $.ajax({
+        url: '/invitees/' + invitee_id,
+        method: "PUT",
+        dataType: "JSON",
+        data: {
+          invitee: {
+            user_id: this.props.userID,
```

Comment:
Misunderstanding, we are passing this in because the user is navigating from an email url link. Not moved to controller.